### ISSUE TITLE
Loki logs do not parse ANSI escape color codes

### ISSUE THREAD

**Issue by avaly** [grafana/loki#253]
I logged messages to Loki which included certain words highlighted with ANSI escape codes.

### What was the expected result?

The message should be parsed with the ANSI escape codes to highlight the text.

### What happened instead?

ANSI escape codes are not parsed/used, instead they are printed out as is.

![image](https://user-images.githubusercontent.com/356100/51908990-5b334080-2406-11e9-8c16-6cd63ba20b63.png)

PS: Opened this in https://github.com/grafana/grafana/issues/15114 but it was pointed out that this might better suited in this repo.

**Comment by DanCech** [comment]
If we want to interpret the codes to color the output then that would need to be implemented in Grafana, all Loki could do would be to strip the codes out, which would mean that no color-coding could be done by any frontend.

**Comment by davkal** [comment]
The question for this repo becomes: Could there be a case where you want to strip the codes? @avaly

**Comment by avaly** [comment]
> Could there be a case where you want to strip the codes?

No, I guess it makes no sense to strip the codes on the ingestion side.

**Comment by tomwilkie** [comment]
I like the idea of Loki being dumb with respect to this kind of thing.  Grafana can and should clean up and/or honor these colors.  Will move issue there.
